README.md

```diff
@@ -25,6 +25,8 @@ npm run dev
 npm run dev -- --open
 ```
 
+If you change the schema of the database you will need to run the `npm run generate` command to create a new migration file. This will create a new file in the `migrations` folder that you can run with the `npm run migrate` command.
```

Comment:
`npx drizzle-kit generate` probably works better but this is good

Reply:
I set up `npm run generate` as an alias for `npx drizzle-kit generate`. Just to keep everything documented in the project.json.